docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -509,7 +509,7 @@
 
               <entry>components</entry>
 
-              <entry></entry>
+              <entry/>
```

Comment:
@g-pan and @JamesDeFabia
you seem to have your editors set up differently.  Jim's changes convert <abc/> to <abc></abc>, and greg's seem to have done the opposite.  
(see https://github.com/hpcc-systems/HPCC-Platform/pull/19390/files) for an example from Jim.
Please can you investigate, and if possible synchronize them.
I'll merge once I see a response to this...

Reply:
yes indeed I do see it, I believe we have gotten out of sync and are on different versions. @JamesDeFabia and I will work offline on this to get back in sync

Reply:
This issue has been addressed, will provide details offline. good to merge?